index.html

```diff
@@ -5632,7 +5632,7 @@ <h2>Text chunk processing</h2>
 
       <p>Decoders running on systems with non-Latin-1 character set encoding should provide character code remapping so that
       Latin-1 characters are displayed correctly. Some systems may not provide all the characters defined in Latin-1. Mapping
-      unavailable characters to a visible notation such as "<code>\nnn</code>" is a good fallback. Character codes 127-255 should
+      unavailable characters to a visible notation such as "<code>\nnn</code>" is a good fallback. Character codes 0x7F-FF should
       be displayed only if they are printable characters on the decoding system. Some systems may interpret such codes as control
```

Comment:
This text is slightly mutant, since it confuses "character codes" with byte values. I would probably rewrite this paragraph thusly:

> Decoders running on systems with a non-Latin-1 [legacy character encoding](https://www.w3.org/TR/i18n-glossary/#def_legacy_character_encoding) should remap character codes so that Latin-1 characters are displayed correctly. Unsupported characters should be replaced with a system-appropriate replacement character (such as U+FFFD REPLACEMENT CHARACTER or U+001A SUB) or mapped to a visible notation such as `\nnn`. Characters should only be displayed if they are printable characters on the decoding system. Some byte values may be interpreted by the decoding system as control characters.

(Note that U+003F QUESTION MARK is a common replacement character in many encodings)

Reply:
Fixed as suggested (including U+003F QUESTION MARK)